packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1296,12 +1296,13 @@ Specifies how lookups for weak passwords are done. Designed to be extensible wit
 | recovery_question | Settings for Security Question Factor | [Recovery Question Factor object](#recovery-question-factor-object) | No       |
 | okta_email        | Settings for Email Factor             | [Email Factor object](#email-factor-object)                         | No       |
 | okta_sms          | Settings for SMS Factor               | [SMS Factor object](#sms-factor-object)                             | No       |
+| okta_call         | Settings for Call Factor              | [Call Factor object](#call-factor-object)                           | No       |
 
 ###### Recovery Question Factor object
 
 | Property   | Description                                         | Data Type                                                                                 | Required |
 | ---        | ---                                                 | ---                                                                                       | ---      |
-| status     | Indicates if the Factor is enabled.                 | `ACTIVE`, `INACTIVE` <ApiLifecycle access="ea" />                                         | Yes      |
+| status     | Indicates if the Factor is enabled.                 | `ACTIVE`, `INACTIVE`                                                                      | Yes      |
 | properties | Configuration settings for Security Question Factor | [Recovery Question Factor Properties object](#recovery-question-factor-properties-object) | No       |
```

Comment:
These are unrelated OIE api fixes but was missing a few details for v1 sspr policy.